client/dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
So for this change (which comes from a change in the webpack), We are explicitly defining the html. Before I had it generating the html and injecting the dependencies it would need. (So when we go to deploy we will have vendor files as well and just simplifies the process IMO). 

What's the benefit for removing that plugin and going to this method?